params/es_params.go

```diff
@@ -1,8 +1,9 @@
 package params
 
 const (
-	EsGetBlobPerByteGas = 3 // Per-byte price for an esGetBlob operation.
-	EsGetBlobBaseGas = 50000 // Base price for an esGetBlob operation.  
+	EsGetBlobPerByteGas uint64 = 1 // Per-byte price for an esGetBlob operation.
```

Comment:
I am not quite sure the motivation of this - if we want to align with the gas cost of EXTCODECOPY, then we should probably use `EsGetBlobPerWordGas` as `3`, and then calculate the gas as `(len + 31) // 32 * 3` as EXTCODECOPY does (i.e., about 0.1 gas per byte).